### ISSUE TITLE
The loss is always 6.9 when I train the default SE-BN-Inception

### ISSUE THREAD

**Issue by 408550969** [hujie-frank/SENet#47]
Hi
There is a strange thing when I train the default SE-BN-Inception,The loss is always 6.9 when I didn't use the pretrained model to train on Imagenet2012(even at iteration 0,the loss is 6.9).And if I use the pretrained model the loss is begin to fall from 9.6.
And I add
weight_filler{
type: "msra"
}
in each convolotion layer(didn't use the pretrained model) ,and the loss can be reduced from 6.9(also 6.9 at iteration 0).
May I ask why this would happen?
